WordPress/Classes/ViewRelated/Plans/PlanListViewController.swift

```diff
@@ -91,6 +91,45 @@ enum PlanListViewModel {
             )
         }
     }
+    
+    func tableFooterView() -> UIView? {
+        switch self {
+        case .Ready:
+            let title = footerTitle
+            let footerView = WPTableViewSectionHeaderFooterView(reuseIdentifier: "ToSFooterView", style: .Footer)
+        
+            footerView.attributedTitle = title
+            footerView.frame.size.height = WPTableViewSectionHeaderFooterView.heightForFooter(title.string, width: footerView.bounds.width)
+            
+            return footerView
+        default:
+            return nil
+        }
+    }
+    
+    private var footerTitle: NSAttributedString {
+        let plainTosText = NSLocalizedString("By checking out, you agree to our fascinating terms and conditions.",
+                                             comment: "TOS label when making a purchase");
+        let tosText = NSLocalizedString("fascinating terms and conditions",
+                                        comment: "'fascinating terms and conditions' should be the same  text that is at the end of 'TOS label when making a purchase'")
+        
+        let attributedText = NSMutableAttributedString(string: plainTosText)
+        
+        // Highlight 'tosText' within the full string
+        let range = (plainTosText as NSString).rangeOfString(tosText, options: .CaseInsensitiveSearch)
+        if range.location != NSNotFound {
+            attributedText.addAttribute(NSForegroundColorAttributeName, value: WPStyleGuide.wordPressBlue(), range: range)
+        }
+        
+        // Add a non-breaking space between the final words so that we don't end up with an orphan
+        // word on the final line when wrapped.
+        let finalSpaceRange = (plainTosText as NSString).rangeOfString(" ", options: .BackwardsSearch)
+        if finalSpaceRange.location != NSNotFound {
+            attributedText.replaceCharactersInRange(finalSpaceRange, withString: "\u{00a0}")
+        }
```

Comment:
Not too happy about this bit. Can we have the nbsp in the string directly? If not, I'd extract this to a `String` method